decred_wallet/view_controller/SendViewController.swift

```diff
@@ -554,17 +552,17 @@ class SendViewController: UIViewController, UITextFieldDelegate,UITextPasteDeleg
         sendCompletedVC.openDetails = { [weak self] in
             guard let `self` = self else { return }
             DispatchQueue.main.async {
-            self.walletAddress.text = nil
-            self.estimateFee.text = "0.00 DCR"
-            self.estimateSize.text = "0 Bytes"
-            self.BalanceAfter.text = "0.00 DCR"
-            self.tfAmount.text = nil
-            if !(self.toAddressContainer.isHidden){
-                self.addQrbtn()
-                self.checkpaste()
-                
-            }
-            self.showDefaultAccount()
+                self.walletAddress.text = nil
+                self.estimateFee.text = "0.00 DCR"
+                self.estimateSize.text = "0 Bytes"
+                self.BalanceAfter.text = "0.00 DCR"
+                self.tfAmount.text = nil
+                if !(self.toAddressContainer.isHidden){
+                    self.addQrbtn()
+                    self.checkpaste()
+                    
```

Comment:
This space is unnecessary. Don't think it makes sense to have an empty line space before a closing curly brace, any closing curly brace - whether for a function, an if statement, a switch statement; any.